wrapper/pre_copy.py

```diff
@@ -991,16 +1014,15 @@ def cleanup(self):
                 disk.proc_qemu.kill()
                 disk.proc_qemu.wait()
             disk.proc_qemu = None
-            if disk.overlay is not None:
-                try:
-                    os.remove(disk.overlay)
-                except FileNotFoundError:
-                    pass
-                except Exception:
-                    error('Cannot remove temporary file "%s", subsequent '
-                          'conversions of the same hose might fail if this '
-                          'file is not removed' % disk.overlay, exception=True)
-            disk.overlay = None
+            path = _get_overlay_path(self._tmp_dir.name, self._vm_name, i)
+            try:
+                os.remove(path)
+            except FileNotFoundError:
+                pass
+            except Exception:
+                error('Cannot remove temporary file "%s", subsequent '
+                      'conversions of the same hose might fail if this '
```

Comment:
I know it was already there, but there seems to be a typo. What "hose" is this talking about?